src/App.js

```diff
@@ -26,12 +39,48 @@ const App = () => {
     getSideDishData(soupUrl, setSoupData);
   }, []);
 
+  const [isVisible, setIsVisible] = useState(false);
+
+  const handleClickMoreCard = () => {
+    setIsVisible(!isVisible);
+  };
+
+  const openBtn = '모든 카테고리 보기';
+  const closeBtn = '열린 카테고리 닫기';
+
   return (
-    <>
+    <StyledApp>
       <Header />
       <MainCard mainCardData={mainCardData} />
-    </>
+      <SubCard
+        dishData={mainDishData}
+        title={'식탁을 풍성하게 하는 정갈한 밑반찬'}
+      />
+      {isVisible && (
+        <SubCard dishData={soupData} title={'정성이 가득한 뜨끈뜨끈한 국물'} />
```

Comment:
1. attribute 스타일도 일관성을 맞춰주면 좋을 것 같습니다
2. 아래도 isVisible에 따라서 보이고 말고가 제어 되는데 fragment로 감싸서 같이 처리 해주면 좋을 것 같네요 
```suggestion
        <SubCard 
            dishData={soupData} 
            title={'정성이 가득한 뜨끈뜨끈한 국물'} 
         />
```

Reply:
1번은 prettier설정으로 인한것이어서 printWidth를 150으로 변경해보았는데 그렇게 되면 이후에 PR보냈을때 모든 파일이 file changed되어서 리뷰하시기 힘드실것이라 판단하여서 일단 그대로 두었습니다. 혹시 file changed 되더라도 바꾸는것이 좋을까요?

Reply:
```suggestion
              {isVisible && (
        <>
          <SubCard
            dishData={soupData}
            title={'정성이 가득한 뜨끈뜨끈한 국물'}
          />
        </>
      )}
      {isVisible && (
        <>
          <SubCard
            dishData={sideDishData}
            title={'모두가 좋아하는 든든한 메인 요리'}
          />
        </>
      )}
```
2번에서 이런식으로 하는것을 말씀하신것일까요? 리뷰의 의도를 잘 파악하지 못한것 같은데 이렇게 한다면 어떤 이유에서 해야할지 궁금합니다. 어떤 키워드로 검색해보면 좋을까요?

Reply:
앗 아닙니따 
1. 스타일 룰에 따른거면 괜찮습니다. 
2. 따로 제어하지 않고 한 번에 감싸서 해도 될 것 같아서 제안드렸습니다. 
```suggestion
     isVisible && (
     <>
       <SubCard 
              dishData={soupData} 
              title={'정성이 가득한 뜨끈뜨끈한 국물'} 
        />
       <SubCard
            dishData={sideDishData}
            title={'모두가 좋아하는 든든한 메인 요리'}
          />
     </>
     )
        
```